### PR TITLE
Fix up references to MS.VS.D.Engine and MS.VS.D.Metadata...

### PR DESCRIPTION
**Pull request by KevinH-MS**
We need to consume the VSSDK nugets for these assemblies in order to
build/run on a machine without VS 2015.

### CODE REVIEW

File: src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpExpressionCompiler.csproj [hunk 1]
```diff
@@ -22,14 +22,9 @@
     <!-- Don't transitively copy output files, since everything builds to the same folder. -->
   </PropertyGroup>
   <ItemGroup Label="File References">
-    <Reference Include="$(OutDir)Microsoft.VisualStudio.Debugger.Engine.dll" />
+    <Reference Include="$(NuGetPackageRoot)\Microsoft.VisualStudio.Debugger.Engine\14.3.25420\lib\portable\Microsoft.VisualStudio.Debugger.Engine.dll" />
```
Comment (KevinH-MS) [inline src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpExpressionCompiler.csproj]: Note:  These explicit references are a temporary hack until the nuget authoring for these packages is fixed for portable.

**Comment by KevinH-MS** [comment]
FYI @dotnet/roslyn-interactive

**Comment by tmat** [comment]
:+1:

**Comment by KevinH-MS** [comment]
retest eta please

**Comment by jasonmalinowski** [comment]
:+1: and yay that the IL hacks are gone. :-)

**Pull request merged**